src/Blueprint/Blueprint/Cmdlets/BlueprintCmdletBase.cs

```diff
@@ -122,9 +123,10 @@ protected override void WriteExceptionError(Exception ex)
         protected void RegisterBlueprintRp(string subscriptionId)
         {
             ResourceManagerClient.SubscriptionId = subscriptionId;
-            var response = ResourceManagerClient.Providers.Register(BlueprintConstants.BlueprintProviderNamespace);
+            var result = ResourceManagerClient.Providers
```

Comment:
@filizt what's the reason for making this change?

Reply:
@cormac I just updated the PR with a new set of changes. The goal is to make sure the Blueprint RP is registered with the subscription in context before going ahead with blueprint/assignment creation. I'll update the description on the PR.